Playbooks/playbook-Tanium_-_Get_saved_question_result.yml

```diff
@@ -0,0 +1,136 @@
+id: Tanium - Get saved question result
```

Comment:
change id and file name to Title-case (see playbook conventions: https://github.com/demisto/internal-content/tree/master/documentation/playbook_conventions)